httpcore5-testing/src/main/java/org/apache/hc/core5/testing/nio/LoggingIOSession.java

```diff
@@ -76,7 +76,7 @@ public Lock getLock() {
     public void enqueue(final Command command, final Command.Priority priority) {
         this.session.enqueue(command, priority);
         if (this.log.isDebugEnabled()) {
-            this.log.debug(this.session + " Enqueued " + command.getClass().getSimpleName() + " with priority " + priority);
+            this.log.debug("{} Enqueued {} with priority {}", this.session, command.getClass().getSimpleName(), priority);
```

Comment:
Enqueued should probably be lowercase, shouldn't it?

Reply:
That's true, I've updated these.